### PR TITLE
Added system context section to Glassbr

### LINKED ISSUE #574: Add system context section to GlassBR example
GlassBR is missing system context section. Update GlassBR SRS by adding system context section into it.

### PR DESCRIPTION
**Pull request by deviprasad135**
Added system context section to Glassbr. Closes #574.

### CODE REVIEW

**Comment by szymczdm** [comment]
This PR is failing our tests. Can you fix that?

File: code/drasil-example/Drasil/GlassBR/Body.hs [hunk 1]
```diff
@@ -119,11 +119,13 @@ mkSRS = RefSec (RefProg intro [TUnits, tsymb [TSPurpose, SymbOrder], TAandA]) :
       [Client gLassBR (S "a" +:+ phrase company
         +:+ S "named Entuitive. It is developed by Dr." +:+ (S $ name mCampidelli)),
       Cstmr gLassBR]) :
-  GSDSec (GSDProg2 [UsrChars [user_characteristics_bullets endUser gLassBR secondYear
+  {--GSDSec (GSDProg2 [UsrChars [user_characteristics_bullets endUser gLassBR secondYear
     undergradDegree civilEng structuralEng glBreakage blastRisk],
-    SystCons [] []]) :
+    SystCons [] []]) :--}
+	Verbatim general_system_description :
```
[the remark this answers is not in the document]
Comment (deviprasad135) [reply code/drasil-example/Drasil/GlassBR/Body.hs]: Sorry for using verbatim. I will fix this.

File: code/drasil-example/Drasil/GlassBR/Body.hs [hunk 2]
```diff
@@ -119,11 +120,13 @@ mkSRS = RefSec (RefProg intro [TUnits, tsymb [TSPurpose, SymbOrder], TAandA]) :
       [Client gLassBR (S "a" +:+ phrase company
         +:+ S "named Entuitive. It is developed by Dr." +:+ (S $ name mCampidelli)),
       Cstmr gLassBR]) :
-  GSDSec (GSDProg2 [UsrChars [user_characteristics_bullets endUser gLassBR secondYear
+  {--GSDSec (GSDProg2 [UsrChars [user_characteristics_bullets endUser gLassBR secondYear
```
[the remark this answers is not in the document]
Comment (deviprasad135) [reply code/drasil-example/Drasil/GlassBR/Body.hs]: I will fix this.

File: code/drasil-example/Drasil/GlassBR/Body.hs [hunk 3]
```diff
@@ -330,22 +326,79 @@ org_of_doc_intro_end = foldl (+:+) EmptyS [(at_start' $ the dataDefn),
 
 {--GENERAL SYSTEM DESCRIPTION--}
 
+general_system_description :: Section
+general_system_description = assembler gLassBR gbSymbMap generalSystemDescriptionSect
+  [sysContext, userCharacteristicSect, systemConstraintSect]
+
```
Comment (szymczdm) [inline code/drasil-example/Drasil/GlassBR/Body.hs]: Some of this code is not being used and should be removed so we're not polluting the examples. Now that you're not using `Verbatim`, you can get rid of the `SubSec`s if they're not explicitly used in the `GSDSec` constructor above.

**Comment by szymczdm** [comment]
@deviprasad135 If this is done, can you fix the conflicts?

**Comment by deviprasad135** [comment]
@szymczdm , I have fixed the conflicts.

**Comment by JacquesCarette** [comment]
@szymczdm the issues I pointed out have been fixed. Your turn.

**Pull request merged**